caffe2/core/macros.h.in

```diff
@@ -15,7 +15,6 @@
 #cmakedefine CAFFE2_THREADPOOL_STATS
 #cmakedefine CAFFE2_USE_ACCELERATE
 #cmakedefine CAFFE2_USE_CUDNN
-#cmakedefine CAFFE2_USE_EIGEN_FOR_BLAS
```

Comment:
Need to check if this is still used internally

Reply:
Reverted